src/ethereum/etcInfo.js

```diff
@@ -0,0 +1,121 @@
+/* global */
+// @flow
+
+import type {
+  EdgeCorePluginOptions,
+  EdgeCurrencyInfo
+} from 'edge-core-js/types'
+
+import { imageServerUrl } from '../common/utils'
+import { makeEthereumBasedPluginInner } from './ethPlugin'
+import type { EthereumSettings } from './ethTypes.js'
+
+const defaultNetworkFees = {
+  default: {
+    gasLimit: {
+      regularTransaction: '21000',
+      tokenTransaction: '200000'
+    },
+    gasPrice: {
+      lowFee: '1000000001',
+      standardFeeLow: '40000000001',
+      standardFeeHigh: '300000000001',
+      standardFeeLowAmount: '100000000000000000',
+      standardFeeHighAmount: '10000000000000000000',
+      highFee: '40000000001'
+    }
+  },
+  '1983987abc9837fbabc0982347ad828': {
+    gasLimit: {
+      regularTransaction: '21002',
+      tokenTransaction: '37124'
+    },
+    gasPrice: {
+      lowFee: '1000000002',
+      standardFeeLow: '40000000002',
+      standardFeeHigh: '300000000002',
+      standardFeeLowAmount: '200000000000000000',
+      standardFeeHighAmount: '20000000000000000000',
+      highFee: '40000000002'
+    }
+  },
+  '2983987abc9837fbabc0982347ad828': {
+    gasLimit: {
+      regularTransaction: '21002',
+      tokenTransaction: '37124'
+    }
+  }
+}
+
+const otherSettings: EthereumSettings = {
+  etherscanApiServers: ['https://blockscout.com/etc/mainnet'],
+  blockcypherApiServers: [],
+  superethServers: [],
+  infuraServers: [],
+  isNestedInfuraParams: false,
+  infuraNeedProjectId: true,
+  uriNetworks: ['ethereumclassic', 'etherclass'],
+  ercTokenStandard: 'ERC20',
+  chainId: 1,
+  hdPathCoinType: 60,
+  checkUnconfirmedTransactions: true,
+  iosAllowedTokens: {
```

Comment:
Remove iosAllowedTokens as it is deprecated.

Reply:
This object appears to be required by the parent object type. For now I just made it an empty object. Unless I should just remove the type and delete from both `ethInfo` and `etcInfo`